app/controllers/api/admin/settings_controller.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+
+class Api::Admin::SettingsController < ApiController
+
+  def test_ldap_connection
+    settings = params[:settings]
```

Comment:
use method 'new', try to use default rails actions

do not accept any params here